apps/api/src/server/applications/application/documents/document.routes.js

```diff
@@ -96,6 +97,43 @@ router.post(
 	asyncHandler(provideDocumentUploadURLs)
 );
 
+router.patch(
+	'/:id/documents/unpublish',
+	/*
+        #swagger.tags = ['Applications']
+        #swagger.path = '/applications/{id}/documents'
```

Comment:
swagger path needs to match actual endpoint, otherwise fails to display correctly in api-docs page.  Also stops the /document endpoint working in api-docs page too.

should say #swagger.path = '/applications/{id}/documents/unpublish'